TypeError: 'Exception' object is not iterable

When I try to run master tip against Antenna running on my local machine, I get this:

```
(M=bc78b) ~/mozilla/socorro-zero/antenna-loadtests> make test -e URL_SERVER=http://localhost:8000/submit
env LDFLAGS="-L/lib" \
    CFLAGS="-I/include" \
    /home/willkg/mozilla/socorro-zero/antenna-loadtests/venv/bin/pip3 install cryptography
Requirement already satisfied: cryptography in ./venv/lib/python3.5/site-packages
Requirement already satisfied: idna>=2.0 in ./venv/lib/python3.5/site-packages (from cryptography)
Requirement already satisfied: pyasn1>=0.1.8 in ./venv/lib/python3.5/site-packages (from cryptography)
Requirement already satisfied: setuptools>=11.3 in ./venv/lib/python3.5/site-packages (from cryptography)
Requirement already satisfied: cffi>=1.4.1 in ./venv/lib/python3.5/site-packages (from cryptography)
Requirement already satisfied: six>=1.4.1 in ./venv/lib/python3.5/site-packages (from cryptography)
Requirement already satisfied: pycparser in ./venv/lib/python3.5/site-packages (from cffi>=1.4.1->cryptography)
/home/willkg/mozilla/socorro-zero/antenna-loadtests/venv/bin/pip3 install -r requirements.txt
Collecting https://github.com/tarekziade/ailoads/archive/master.zip (from -r requirements.txt (line 4))
  Downloading https://github.com/tarekziade/ailoads/archive/master.zip
     / 40kB 1.1MB/s
Requirement already satisfied: six in ./venv/lib/python3.5/site-packages (from -r requirements.txt (line 1))
Requirement already satisfied: requests in ./venv/lib/python3.5/site-packages (from -r requirements.txt (line 2))
Requirement already satisfied: flake8 in ./venv/lib/python3.5/site-packages (from -r requirements.txt (line 3))
Collecting aiohttp (from molotov==0.4->-r requirements.txt (line 4))
  Downloading aiohttp-1.2.0-cp35-cp35m-manylinux1_x86_64.whl (155kB)
    100% |████████████████████████████████| 163kB 1.6MB/s 
Requirement already satisfied: statsd in ./venv/lib/python3.5/site-packages (from molotov==0.4->-r requirements.txt (line 4))
Collecting urwid (from molotov==0.4->-r requirements.txt (line 4))
  Downloading urwid-1.3.1.tar.gz (588kB)
    100% |████████████████████████████████| 593kB 1.9MB/s 
Collecting humanize (from molotov==0.4->-r requirements.txt (line 4))
  Downloading humanize-0.5.1.tar.gz
Collecting redis (from molotov==0.4->-r requirements.txt (line 4))
  Downloading redis-2.10.5-py2.py3-none-any.whl (60kB)
    100% |████████████████████████████████| 61kB 2.9MB/s 
Requirement already satisfied: pycodestyle<2.3.0,>=2.0.0 in ./venv/lib/python3.5/site-packages (from flake8->-r requirements.txt (line 3))
Requirement already satisfied: pyflakes!=1.2.0,!=1.2.1,!=1.2.2,<1.4.0,>=0.8.1 in ./venv/lib/python3.5/site-packages (from flake8->-r requirements.txt (line 3))
Requirement already satisfied: mccabe<0.6.0,>=0.5.0 in ./venv/lib/python3.5/site-packages (from flake8->-r requirements.txt (line 3))
Collecting chardet (from aiohttp->molotov==0.4->-r requirements.txt (line 4))
  Downloading chardet-2.3.0.tar.gz (164kB)
    100% |████████████████████████████████| 174kB 6.7MB/s 
Collecting multidict>=2.1.4 (from aiohttp->molotov==0.4->-r requirements.txt (line 4))
  Downloading multidict-2.1.4-cp35-cp35m-manylinux1_x86_64.whl (357kB)
    100% |████████████████████████████████| 358kB 2.6MB/s 
Collecting async-timeout>=1.1.0 (from aiohttp->molotov==0.4->-r requirements.txt (line 4))
  Downloading async_timeout-1.1.0-py3-none-any.whl
Collecting yarl>=0.8.1 (from aiohttp->molotov==0.4->-r requirements.txt (line 4))
  Downloading yarl-0.8.1-cp35-cp35m-manylinux1_x86_64.whl (137kB)
    100% |████████████████████████████████| 143kB 2.2MB/s 
Building wheels for collected packages: urwid, humanize, chardet
  Running setup.py bdist_wheel for urwid ... done
  Stored in directory: /home/willkg/.cache/pip/wheels/e7/bb/9d/d7f7fcc9b3aeae20cf4685667635002415629e6395ff2dda7d
  Running setup.py bdist_wheel for humanize ... done
  Stored in directory: /home/willkg/.cache/pip/wheels/d4/80/38/cfbfd95752f71f3812505b948b43383ddc99eedf835fc13b09
  Running setup.py bdist_wheel for chardet ... done
  Stored in directory: /home/willkg/.cache/pip/wheels/28/8c/bf/a69199bd4901d84e13362f95a9ea7bc9a691fed2d655a90bc4
Successfully built urwid humanize chardet
Installing collected packages: chardet, multidict, async-timeout, yarl, aiohttp, urwid, humanize, redis, molotov
  Running setup.py install for molotov ... done
Successfully installed aiohttp-1.2.0 async-timeout-1.1.0 chardet-2.3.0 humanize-0.5.1 molotov-0.4 multidict-2.1.4 redis-2.10.5 urwid-1.3.1 yarl-0.8.1
bash -c "URL_SERVER=http://localhost:8000/submit /home/willkg/mozilla/socorro-zero/antenna-loadtests/venv/bin/ailoads -v -d 30"
[th]Traceback (most recent call last):
  File "/home/willkg/mozilla/socorro-zero/antenna-loadtests/venv/bin/ailoads", line 11, in <module>
    load_entry_point('ailoads==0.1', 'console_scripts', 'ailoads')()
  File "/home/willkg/mozilla/socorro-zero/antenna-loadtests/venv/lib/python3.5/site-packages/ailoads/run.py", line 71, in main
    return run(args)
  File "/home/willkg/mozilla/socorro-zero/antenna-loadtests/venv/lib/python3.5/site-packages/ailoads/run.py", line 85, in run
    for ok, failed in res:
TypeError: 'Exception' object is not iterable
Makefile:56: recipe for target 'test' failed
make: *** [test] Error 1
2 (M=bc78b) ~/mozilla/socorro-zero/antenna-loadtests> 
```

Am I doing something wrong? If not, any idea what's going on.

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,5 +1,5 @@
 # Mozilla Load-Tester
-FROM stackbrew/debian:testing
+FROM ubuntu
 
 RUN \
     apt-get update; \
@@ -9,7 +9,6 @@ RUN \
     cd /home/loads; \
     pip3 install virtualenv; \
     make build -e PYTHON=python3.5; \
-    apt-get remove -y -qq git build-essential make python3-pip python3-venv libssl-dev libffi-dev; \
     apt-get autoremove -y -qq; \
     apt-get clean -y
 

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,18 +1,19 @@
 # Mozilla Load-Tester
-FROM ubuntu
+FROM python:3.5-slim
+WORKDIR /home/loads/antenna-loadtests
 
-RUN \
-    apt-get update; \
-    apt-get install -y python3-pip python3-venv git build-essential make; \
-    apt-get install -y python3-dev libssl-dev libffi-dev; \
-    git clone -b dev https://github.com/rpappalax/antenna-loadtests.git /home/loads; \
-    cd /home/loads; \
-    pip3 install virtualenv; \
-    make build -e PYTHON=python3.5; \
-    apt-get autoremove -y -qq; \
-    apt-get clean -y
+RUN apt-get update
+RUN apt-get install -y git
+RUN pip3 install -U pip
+RUN pip3 install virtualenv;
+RUN virtualenv venv -p python3;
+RUN . ./venv/bin/activate
+# RUN venv/bin/pip3 install -r requirements.txt;
+RUN venv/bin/pip3 install git+git://github.com/tarekziade/molotov.git@284e47562e17523b4504047dd05b570938596d58
+# RUN apt-get autoremove -y -qq
+# RUN apt-get clean -y
 
-WORKDIR /home/loads
+ADD . /home/loads/antenna-loadtests
 
 # run the test
-CMD venv/bin/ailoads -v -d $TEST_DURATION -u $CONNECTIONS
+CMD venv/bin/ailoads -v -d $TEST_DURATION -u $TEST_CONNECTIONS

--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,4 @@
-PROJECT="antenna"
+PROJECT="antenna-loadtests"
 OS := $(shell uname)
 HERE = $(shell pwd)
 PYTHON = python3
@@ -52,19 +52,18 @@ configure: build
 
 
 #bash -c "source loadtest.env && URL_SERVER=$(URL_SERVER) $(BIN)/ailoads -v -d 30"
-test: build
-	bash -c "URL_SERVER=$(URL_SERVER) $(BIN)/ailoads -v -d 30"
-	$(BIN)/flake8 loadtest.py
+test:
+	bash -c "URL_SERVER=$(URL_SERVER) ailoads -v -d 30"
 
 test-heavy: build
 	bash -c "source loadtest.env && URL_SERVER=$(URL_SERVER) $(BIN)/ailoads -v -d 300 -u 10"
 
 
 docker-build:
-	docker build -t $(PROJECT)/loadtest .
+	docker build -t $(PROJECT) .
 
 docker-run:
-	bash -c "source loadtest.env; docker run -e TEST_DURATION=30 -e CONNECTIONS=4 $(PROJECT)/loadtest"
+	bash -c "source loadtest.env; docker run -e TEST_DURATION=30 -e CONNECTIONS=4 $(PROJECT)"
 
 docker-export:
 	docker save "$(PROJECT)/loadtest:latest" | bzip2> "$(PROJECT)-latest.tar.bz2"

--- a/loads.tpl
+++ b/loads.tpl
@@ -11,7 +11,7 @@ cat > loads.json <<EOF
       "description": "4 boxes",
       "steps": [
         {
-          "name": "Shavar: Test Cluster",
+          "name": "Antenna",
           "instance_count": 4,
           "instance_region": "us-east-1",
           "instance_type": "m3.medium",

--- a/loadtest.py
+++ b/loadtest.py
@@ -12,7 +12,7 @@ if DEBUG:
     utils._log_everything()
 
 
-@scenario(0)
+@scenario(100)
 def test_crash_100k_compressed():
     size = (100 * 1024)
     raw_crash, dumps = utils.generate_sized_crashes(size)

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,4 +1,3 @@
 six
 requests
-flake8
-https://github.com/tarekziade/ailoads/archive/master.zip
+git+git://github.com/tarekziade/molotov.git@284e47562e17523b4504047dd05b570938596d58

--- a/send_crash_report.sh
+++ b/send_crash_report.sh
@@ -1,0 +1,18 @@
+#!/bin/bash
+
+# Sends a fake crash report to the host specified in $HOST 
+# defaulting to http://localhost:8000 .
+HOST=$1
+if [ -z "${HOST}" ]; then
+	    HOST="http://localhost:8000"
+    fi
+
+    URL="${HOST}/submit"
+
+    curl -v -H 'Host: crash-reports' \
+	    -F 'uuid=a448814e-16dd-45fb-b7dd-b0b522161010' \
+  	    -F 'ProductName=Firefox' \
+	    -F 'Version=1' \
+	    -F upload_file_minidump2=@small.dmp \
+				     "$URL"
+

